python/cuml/dask/common/comms.py

```diff
@@ -308,10 +312,11 @@ def _func_store_initial_state(nworkers, sessionId, uniqueId, wid):
         for ep in worker_state(sessionId)["ucp_eps"]:
             if ep is not None:
                 if not ep.closed():
-                    ep.close()
+                    ep.abort()
```

Comment:
Why this change?

Reply:
Once we get to this point, we shouldn't be expecting any further messages so we want the endpoints killed immediately without waiting for them to flush.

Reply:
Makes sense